testpackage/tests/Magnetosphere_polar_small/Magnetosphere_polar_small.cfg

```diff
@@ -99,6 +99,7 @@ precedence = 2
 
 [proton_conductingsphere]
 rho = 1.0e6
+T=100000.0
```

Comment:
this should be 5e5, according to line 137-138

Reply:
Oh, true. My bad.